packages/flutter_test/pubspec.yaml

```diff
@@ -34,7 +34,6 @@ dependencies:
   characters: 1.3.0 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   collection: 1.17.2 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   js: 0.6.7 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  matcher: 0.12.15 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
This looks off - `matcher` should remain a dependency of `flutter_test`.

Reply:
ahh, good catch. I guess flutter_test now needs to directly depend on it, rather than transitively via `test_api`.